demo/lib/demo_web/live/user_live.ex

```diff
@@ -24,7 +24,7 @@ defmodule DemoWeb.UserLive do
   @impl Backpex.LiveResource
   def can?(_assigns, _action, _item), do: true
 
-  def item_query(query, live_action, _assigns) when live_action in [:index, :resource_action] do
+  def item_query(query, live_action, _assigns) when live_action in [:index, :show, :edit, :resource_action] do
```

Comment:
```suggestion
  def item_query(query, live_action, _assigns) do
```

I think we can remove the guard because it covers all possible actions (?) On `new` it has no effect anyway.